tests/integration_tests/common.py

```diff
@@ -17,3 +20,32 @@ def wait_port(port_num, timeout):
         in_use = is_port_in_use(port_num)
 
     return in_use
+
+def prepare_config(config):
```

Comment:
Does this need to be here, can't it just be replaced by the generate config functionality in the config wizard ? + some changes to the json file that generates ?

Reply:
I think it needed here, because this function only for integration tests. This function is add two changes to current config:
- switch off all integrations, except current tested (now hardcoded 'default_mariadb', but this function wiull use in clichhouse tests too)
- change predictors and datasources dirs to some temp dir